versioned/storage/store/src/main/java/org/projectnessie/versioned/storage/versionstore/BaseCommitHelper.java

```diff
@@ -387,6 +387,17 @@ private void validateNamespacesExistForContentKeys(
     }
   }
 
+  static ObjId idForExpectedContent(StoreKey key, StoreIndex<CommitOp> headIndex) {
```

Comment:
seems like all callers use `el.key()` for the 1st param, maybe this should happen inside the method?